frontend/src/plugin-extensions/docs/acm-extensions.md

```diff
@@ -0,0 +1,44 @@
+# ACM Plugin Extension Types
```

Comment:
Are you creating this doc all by yourself?

Reply:
yes

Reply:
This doc should be generated for now. Let's remove this. We will take up doc generation in a different story altogether.